ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -600,6 +601,11 @@ default void storageHandlerCommit(Properties commitProperties, Operation operati
     throw new UnsupportedOperationException();
   }
 
+  default void storageHandlerCommit(Properties commitProperties, Operation operation, TaskAttemptContext context)
```

Comment:
1. javadoc is missing
2. why do we need this?

Reply:
can we execute the commit directly on the iceberg table?

Reply:
copying response from another comment:
```we cannot depend on iceberg in hive-exec in compile-time, there is no iceberg* dependency in hive-exec/pom.xml, that's what's hidden by the storagehandler layer, I think, even if I really like iceberg's api, I'm limited to use the storagehandler
```